testpmd-container-app/testpmd/Dockerfile

```diff
@@ -44,15 +44,27 @@ RUN go mod init webserver.go
 RUN GOOS=linux CGO_ENABLED=0 go build -a -o webserver .
 
 ## testpmd image
-FROM quay.io/centos/centos:stream8
+# Use ubi8/ubi as image
+FROM registry.access.redhat.com/ubi8/ubi:latest
 
-# Required libraries and debugging tools
+# Install prerequisite packages from UBI repos
 RUN dnf install -y \
-    epel-release \
     libibverbs \
     logrotate \
-    numactl \
     python3 \
+    && dnf clean all
+
+# Install prerequisite packages from CentOS Stream repos
+COPY files/centos8-base.repo /etc/yum.repos.d/centos8-base.repo
+COPY files/centos8-app.repo /etc/yum.repos.d/centos8-app.repo
+COPY files/centos8-extras.repo /etc/yum.repos.d/centos8-extras.repo
+
+RUN dnf install -y \
+    --enablerepo=centos8-base \
+    --enablerepo=centos8-app \
+    --enablerepo=centos8-extras \
+    epel-release \
+    numactl \
```

Comment:
could we install these packages from rhel repo?

Reply:
For the moment, I'm going to merge this and I'll create a Jira card to explore this. For the ubi9 case, I've checked it manually and I foresee the same issue we're having now: we need to get some packages from external repos too. For the moment, I decided to move to ubi8 to avoid known issues with OCP 4.12 and RHEL9-related images.